src/main/java/org/jitsi/videobridge/Content.java

```diff
@@ -250,36 +250,76 @@ public RtpChannel createRtpChannel(String channelBundleId,
                                        Boolean initiator,
                                        RTPLevelRelayType rtpLevelRelayType)
         throws Exception
+    {
+        return createRtpChannel(
+                channelBundleId, transportNamespace,
+                initiator, rtpLevelRelayType,
+                false /* octo */);
+    }
+
+    /**
+     * Initializes a new <tt>RtpChannel</tt> instance and adds it to the list of
+     * <tt>RtpChannel</tt>s of this <tt>Content</tt>. The new
+     * <tt>RtpChannel</tt> instance has an ID which is unique within the list of
+     * <tt>RtpChannel</tt>s of this <tt>Content</tt>.
+     *
+     * @param channelBundleId the ID of the channel-bundle that the created
+     * <tt>RtpChannel</tt> is to be a part of (or <tt>null</tt> if it is not to
+     * be a part of a channel-bundle).
+     * @param transportNamespace transport namespace that will used by new
+     * channel. Can be either {@link IceUdpTransportPacketExtension#NAMESPACE}
+     * or {@link RawUdpTransportPacketExtension#NAMESPACE}.
+     * @param initiator the value to use for the initiator field, or
+     * <tt>null</tt> to use the default value.
+     * @param rtpLevelRelayType
+     * @return the created <tt>RtpChannel</tt> instance.
+     * @param octo whether to create a regular channel or an Octo channel.
+     * @throws Exception
+     */
+    public RtpChannel createRtpChannel(String channelBundleId,
+                                       String transportNamespace,
+                                       Boolean initiator,
+                                       RTPLevelRelayType rtpLevelRelayType,
+                                       boolean octo)
+        throws Exception
     {
         RtpChannel channel;
 
         synchronized (channels)
         {
             String id = generateUniqueChannelID();
 
-            switch (getMediaType())
+            if (octo)
+            {
+                channel = new OctoChannel(this, id);
```

Comment:
we can't use `getMediaType()` for `OctoChannel` because it contains all media types, is that right?  If so, how does it end up being created within a single `Content`? Won't it need to be in both?